savant/tests/test_basic.py

```diff
@@ -0,0 +1,19 @@
+import unittest
```

Comment:
Actually, I noticed there are other commits as well. You should revert/remove everything from `6e27a89` forward, and break those into new PRs.

Next time we video chat, I'll show you why I need these split.